config/sections/mixins/parent.php

```diff
@@ -34,7 +34,7 @@
             }
 
             if ($parent === null) {
-                return $this->model;
+                $parent = $this->model;
```

Comment:
Does this change make any difference?

Reply:
No, it doesn't. I had a different kind of fix in the line below that needed that change, but forgot to remove it.